deployment/common/changeset/example/datastore/exemplar_deploy_link_token.go

```diff
@@ -0,0 +1,157 @@
+package example
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/Masterminds/semver/v3"
+
+	chain_selectors "github.com/smartcontractkit/chain-selectors"
+
+	ds "github.com/smartcontractkit/chainlink-deployments-framework/datastore"
+
+	"github.com/smartcontractkit/chainlink-evm/gethwrappers/shared/generated/link_token"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	exemplarMd "github.com/smartcontractkit/chainlink/deployment/exemplar/metadata"
+)
+
+// ExemplarDeployLinkToken implements the deployment.ChangeSetV2 interface
+var _ deployment.ChangeSetV2[uint64] = ExemplarDeployLinkToken{}
+
+// ExemplarDeployLinkToken is an example changeset that deploys the LinkToken contract to an EVM chain and updates the
+// environment datastore using the exemplar metadata types.
+type ExemplarDeployLinkToken struct{}
+
+// VerifyPreconditions checks if the chainSelector is a valid EVM chain selector.
+func (cs ExemplarDeployLinkToken) VerifyPreconditions(_ deployment.Environment, chainSelector uint64) error {
+	fam, err := chain_selectors.GetSelectorFamily(chainSelector)
+	if err != nil {
+		return fmt.Errorf("failed to get chain selector family: %w", err)
+	}
+
+	if fam != chain_selectors.FamilyEVM {
+		return fmt.Errorf("invalid chain selector for EVM: %d", chainSelector)
+	}
+
+	return nil
+}
+
+// Apply deploys the LinkToken contract to the specified EVM chain and updates the environment datastore.
+func (cs ExemplarDeployLinkToken) Apply(e deployment.Environment, chainSelector uint64) (deployment.ChangesetOutput, error) {
+	// Read contents of the environment datastore. Since it is loaded using the DefaultMetadata types, it needs to be converted
```

Comment:
```suggestion
	// Read contents of the environment datastore. Since it is loaded using the DefaultMetadata types, it needs to be converted
```

DefaultMetadata does not exists anymore right?

Reply:
this comment is not needed anymore as I am removing the whole transform step